lakefile.lean

```diff
@@ -12,6 +12,12 @@ lean_exe mlirnatural {
   root := `SSA.MLIRNatural
 }
 
+
+lean_exe opt {
+  root := `SSA.Projects.InstCombine.LLVM.Opt
+  supportInterpreter := true
```

Comment:
I am curious why we need the `supportInterpreter` here.

Reply:
that's because we use the Lean frontend in our code (we call the Lean parser), so that it also links it.

Reply:
Maybe add a comment explaining this.